packages/pn-personafisica-webapp/src/components/Contacts/__test__/CourtesyContacts.test.tsx

```diff
@@ -176,16 +213,16 @@ describe('CourtesyContacts Component', async () => {
       .reply(204);
     const result = render(
       <DigitalContactsCodeVerificationProvider>
-        <CourtesyContactItem type={CourtesyFieldType.PHONE} value={defaultPhoneAddress!.value} />
+        <CourtesyContacts contacts={digitalCourtesyAddresses} />
       </DigitalContactsCodeVerificationProvider>,
       {
         preloadedState: {
           contactsState: { digitalAddresses: [defaultPhoneAddress] },
         },
       }
     );
-    const editButton = result.getByRole('button', { name: 'button.modifica' });
-    fireEvent.click(editButton);
+    const editButton = result.getAllByRole('button', { name: 'button.modifica' });
```

Comment:
don't use getAllByRole. check how it has been done in "override an existing email with a new one"